pages/installing-packages.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: page
+title: Installing Python packages with conda-forge
+permalink: /installing-packages
+nav_order: 35
+---
+
+# Avoiding the spaghetti installation
+
+Many ways to install Python and Python packages have been developed over the years, and not all of them are compatible.
+
+If you're having trouble installing Scikit-HEP packages (e.g. `pip install` fails with an error or you get an `ImportError`/`ModuleNotFoundError`), try following the instructions below to set up a conda-forge environment using `mamba`
+
+![A mess of Python environments](https://imgs.xkcd.com/comics/python_environment.png)
+
+The instructions are at the bottom of this page, after describing what the installation is and what it will do to your computer.
```

Comment:
I would start with the instructions and then the explanation.

Reply:
I thought about that. Maybe I'm a cautious computer user, but I want to know what an installation is going to do before I even consider it*. Knowing that it's easy to back out of a bad conda installation would put me at ease, and I'd want to know about that first. Of course, someone could read the page out of order, too.

@henryiii had a good idea about putting some "too much information" sections into a drop-down. Maybe all of these informational sections can go into drop-downs?

(* Back when everything was installed through `./configure && make && make install`, I kept a paper notebook of exactly what was done to the computer, so that I could start over from scratch and reproduce it. I had to use that notebook several times!)

Reply:
> Maybe I'm a cautious computer user, 

Yes ;)  

To be fair, this is the order I also would want this information and the order I would write it in, but I keep being told that (almost) no one else wants to read it in that order.

Reply:
Done.